google-cloud-translate/samples/README.md

```diff
@@ -17,43 +17,36 @@ Authentication is typically done through [Application Default Credentials](https
 environment has credentials. You have a few options for setting up
 authentication:
 
-1.  When running locally, use the [Google Cloud SDK](https://cloud.google.com/sdk/)
+1. When running locally, use the [Google Cloud SDK](https://cloud.google.com/sdk/)
 
-        gcloud auth application-default login
+       gcloud auth application-default login
 
-1.  When running on App Engine or Compute Engine, credentials are already set-up.
-    However, you may need to configure your Compute Engine instance with
-    [additional scopes](https://cloud.google.com/compute/docs/authentication#using).
+1. When running on App Engine or Compute Engine, credentials are already set-up.
+However, you may need to configure your Compute Engine instance with
+[additional scopes](https://cloud.google.com/compute/docs/authentication#using).
 
-1.  You can create a [Service Account key file](https://cloud.google.com/docs/authentication#service_accounts).
-    This file can be used to authenticate to Google Cloud Platform services from
-    any environment. To use the file, set the `GOOGLE_APPLICATION_CREDENTIALS`
-    environment variable to the path to the key file, for example:
+1. You can create a [Service Account key file](https://cloud.google.com/docs/authentication#service_accounts)
+. This file can be used to authenticate to Google Cloud Platform services from
```

Comment:
Move this orphaned period back to the previous line.

Reply:
Thanks, done.